### ISSUE TITLE
Confusion about the pre-training cross-attention's K and V

### ISSUE THREAD

**Issue by wangbxj1234** [yformer/EfficientSAM#46]
'keys and values derive from both unmasked features from encoder and masked features.'        But I have no idea where do the  masked features come from?   what do the 'masked features' refer to?     And how did you merge these with the unmasked features from the encoder?    I can't have a clear idea from the paper.
Thank you a lot!
